commands/codebuild_cli.py

```diff
@@ -10,12 +10,8 @@
 
 
 def import_pat(pat, client):
-
     response = client.import_source_credentials(
-        token=pat,
-        serverType='GITHUB',
-        authType='PERSONAL_ACCESS_TOKEN',
-        shouldOverwrite=True
+        token=pat, serverType="GITHUB", authType="PERSONAL_ACCESS_TOKEN", shouldOverwrite=True
```

Comment:
What if we wanted it on multiple lines because we find that easier to grok? It will just get automatically put onto one line?

Reply:
You can add the comment `# fmt: off` at the top of the block and then turn the formatter back on at the end of the block with `# fmt: on`.

```suggestion
        # fmt: off
        token=pat,
        serverType="GITHUB",
        authType="PERSONAL_ACCESS_TOKEN",
        shouldOverwrite=True
        # fmt: on
```

Reply:
Here's a direct link to the docs: https://black.readthedocs.io/en/stable/the_black_code_style/current_style.html#code-style

Reply:
Yes, I think so. But, if you really want to keep it on multiple lines and avoid the `fmt: on/off` comments, then you need to append a trailing comma, and it will not automatically put it onto one line. The docs also recommend this approach.

```suggestion
        token=pat,
        serverType="GITHUB",
        authType="PERSONAL_ACCESS_TOKEN",
        shouldOverwrite=True,
```

Reference: https://black.readthedocs.io/en/stable/the_black_code_style/current_style.html#trailing-commas